events/events_network.c

```diff
@@ -451,7 +451,7 @@ events_network_get(void)
 			break;
 		}
 
-		/* Are we ready for reading? */
+		/* Are we ready for writing? */
```

Comment:
The "reading/writing" was changed in
        2017-01-22 Switch events_network from select to poll
        91f42f7f50f9e1c61ada00f4724b0d761a4b56c0
but it should have remained "writing".